src/components/Book.js

```diff
@@ -1,61 +1,49 @@
-import PropTypes from 'prop-types';
-import { useDispatch } from 'react-redux';
-import { checkStatus, removeBook } from '../redux/books/booksSlice';
+import { useEffect } from 'react';
+import { useDispatch, useSelector } from 'react-redux';
+import booksDisplay from './displayBooks';
 import Form from './Form';
-import books from './books';
+import removeBook from './removeBook';
 
-const Books = () => (
-  <div>
-    {books.map((book) => (
-      <Book title={book.title} category={book.category} author={book.author} progress="64%" chapter="Chapter 17" id={book.id} key={book.id} />
-    ))}
-    <Form />
-  </div>
-);
-
-const Book = ({
-  category, title, author, progress, chapter, id,
-}) => {
+const Books = () => {
   const dispatch = useDispatch();
+  const books = useSelector((state) => (state.books.books));
+  useEffect(() => {
+    dispatch(booksDisplay());
+  }, [books]);
 
   return (
-    <div className="book">
-      <div className="book-details">
-        <p>{category}</p>
-        <h2 className="title">{title}</h2>
-        <p className="author">{author}</p>
-        <ul className="book-action">
-          <li>Comments</li>
-          <button type="button" onClick={() => dispatch(removeBook({ id }))}>Remove</button>
-          <li>Edit</li>
-          <button type="button" className="status-btn" onClick={() => dispatch(checkStatus({ id }))}>Status</button>
-        </ul>
-      </div>
-      <div>
-        <div className="oval" />
-        <div>
-          <h3 className="per-completed">{progress}</h3>
-          <p className="completed">completed</p>
+    <div>
+      {books.map((book) => (
+        <div className="book" key={book.key}>
+          <div className="book-details">
+            <p className="book-category">{book.category}</p>
+            <h2 className="title">{book.title}</h2>
+            <p className="author">{book.author}</p>
+            <ul className="book-action">
+              <li>Comments</li>
+              <li><button type="button" id={book.key} onClick={() => dispatch(removeBook(book.key))}>Remove</button></li>
+              <li>Edit</li>
+            </ul>
+          </div>
+          <div className="per-div">
+            <div className="oval" />
+            <div>
+              <h3 className="per-completed">0%</h3>
+              <p className="completed">Completed</p>
```

Comment:
Good job taking into account some of the requested changes, however, some of the font families are still not in coherence with the zeplin template. Recommend making this change to make your design much similar to the template.

KEY: the blue highlights should be using RobotoSlab as font face and not sans-serif

You have set the correct color (#121212) on the nav links; however, you change is not taking effect because you added the color to the `li` (.nav-items li) and not the anchor tags themselves. Kindly update this properly to give the user a good experience as the default color on a tag is not coherent with the apps color scheme.

<img width="812" alt="fix" src="https://github.com/flemton/bookstore/assets/84930355/b20bf6e5-8b27-4b1d-80ea-f1431b726c03">

Reply:
Thank you very much for notifying me about this sir